src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -53,6 +53,23 @@ namespace {
 
         return label.getElement();
     };
+
+    bool checkMessage(const QString &userName, MessagePtr message)
```

Comment:
nit: can this have a more descriptive name?

Reply:
Fixed in 91c3889 :+1: